vaadin-spring/src/main/java/com/vaadin/flow/spring/security/VaadinSecurityConfigurer.java

```
@@ -426,8 +426,8 @@ public VaadinSecurityConfigurer enableAuthorizedRequestsConfiguration(
      * Configures the access rule for any request not matching other configured
      * rules.
      * <p>
-     * The default rule is to require authentication, which is the equivalent of
-     * passing {@link AuthorizedUrl#authenticated()} to this method.
+     * The default rule is to restrict access, which is the equivalent of
+     * passing {@link AuthorizedUrl#denyAll()} to this method.
```

In existing applications, some routes may unintentionally be missing an explicit `authenticated()` rule. These now silently fail with a 403. I ran into this with images, for example. It could be helpful to log a debug message when an authenticated user is trying to access an _existing_ route that has become "denyAll".